gnmi_cli_py/py_gnmicli.py

```diff
@@ -275,6 +306,15 @@ def _get_val(json_value):
       raise JsonReadError('Error while loading JSON: %s' % str(e))
     val.json_ietf_val = json.dumps(set_json).encode()
     return val
+  elif '$' in json_value:
+    try:
+      proto_bytes = six.moves.builtins.open(json_value.strip('$'), 'rb').read()
+    except (IOError, ValueError) as e:
+      raise Exception('Error while loading proto: %s' % str(e))
+    val.proto_bytes = proto_bytes
+    return val
+  elif '#' in json_value:
```

Comment:
>'#' in json_value [](http://example.com/codeflow?start=7&length=17)

Is it a new feature in command line? I do not see any usage text or code comment, so hard to read.

Reply:
"Use '#' to indicate empty value" is kind of hacky to me.
could you just use empty python string to "to indicate empty value"?

Reply:
Updated